src/main/java/com/javadiscord/javabot/service/economy/dao/TransactionRepository.java

```diff
@@ -1,7 +1,7 @@
 package com.javadiscord.javabot.service.economy.dao;
 
-import com.javadiscord.javabot.utils.StringResourceCache;
 import com.javadiscord.javabot.service.economy.model.Transaction;
+import com.javadiscord.javabot.utils.StringResourceCache;
```

Comment:
There's no reason for adding a useless newline, and rearranging the imports to no longer be in alphabetical order.